prometheus_api_client/prometheus_connect.py

```diff
@@ -64,6 +64,19 @@ def __init__(
         self._session = requests.Session()
         self._session.mount(self.url, HTTPAdapter(max_retries=retry))
 
+    def check_prometheus_connection(self, params: dict = None) -> bool:
+        """Check database metrics (Prometheus) connection."""
+        response = self._session.get(
+            "{0}/".format(self.url),
```

Comment:
`{0}/` should work fine I think, but do you think it would be better to use one of the [status](https://prometheus.io/docs/prometheus/latest/querying/api/#status) endpoints for this?

Reply:
The method introduced targets the availability of the endpoint itself, so without that being available, you cannot ask for other information, including the one provided https://prometheus.io/docs/prometheus/latest/querying/api/#status. In that case we could have another method if required.

I thinkg we can leave `{0}` in general for checking the availability and connection to a Prometheus/Thanos endpoint. WDYT? @4n4nd

Reply:
aaah yes, this would work for Thanos too :+1: